### DISCUSSION TITLE
Format Date (not 'formatString')

### DISCUSSION THREAD

**Discussion by rich1n** [gatsbyjs/gatsby#33166] [Help] [answered]
My blogs post from strapi only have two keys for dates: 'updatedAt' and 'published_at'; The result is 

```
2021-09-12T17:32:49.031Z
```

How can I do to parse this data in something like 'september, 20' or '2021 july, 19'

I tried momentJs but only send actual date. all my blog post have the same day in the date.

**Comment by rich1n** [comment]
Done I was doing the following code
```
moment({updatedAt}).format("MMMM DD YYYY")
```

but the good was this

```
moment(blog.updatedAt).locale('es').format('DD MMMM YYYY')
```

**Reply by KyleAMathews** [reply]
You can format the strings in your grapqhql query. This means you don't have to load the heavy moment library into your pages.

**Reply by rich1n** [reply]
I installed the correct plugin but I need to have 2 dates with different style (18 Feb for example) how can I do this (with the locale function) without moment.js?

```
                  <h1>
                      { moment(blog.updatedAt).format("DD") }
                    </h1>
                    <span>
                      {isDefaultLang + '.' ? 
                      moment(blog.updatedAt).locale('es').format("MMM") 
                      : 
                      moment(blog.updatedAt).format("MMM") }
                    </span>
                  </div>
```

**Comment by rich1n** [comment, marked as answer]
I found the reason it didn't work; I have installed gatsby-source-graphql and not gatsby-source-strapi...
